playbooks/cluster_create_post_install.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: post-install tasks
+  hosts: localhost
+  tasks:
+  - name: Get instance facts
+    ec2_instance_facts:
+        aws_access_key: "{{ aws_access_key }}"
+        aws_secret_key: "{{ aws_secret_key }}"
+        region: "{{ oo_sublocation }}"
+        filters: "tag:clusterid={{ oo_clusterid }},tag:host-type=master"
+    register: facts
+
+  - name: add first master to firstMaster group
```

Comment:
You should be able to use the `run_once` param here, rather than pulling out the first master host. See https://docs.ansible.com/ansible/latest/user_guide/playbooks_delegation.html#run-once